testsuite/features/secondary/min_centos_salt.feature

```diff
@@ -1,58 +1,58 @@
-# Copyright (c) 2017-2019 SUSE LLC
+# Copyright (c) 2017-2020 SUSE LLC
 # Licensed under the terms of the MIT license.
 #
-# 1) delete CentOS SSH minion and register as Centos minion
+# 1) delete CentOS minion and register as Centos SSH minion
 # 2) run a remote command
 # 3) try an openscap scan
-# 4) delete CentOS minion client and register as Centos SSH minion
+# 4) delete CentOS SSH minion client and register as Centos minion
 
-Feature: Be able to bootstrap a CentOS minion and do some basic operations on it
+Feature: Bootstrap a SSH-managed CentOS minion and do some basic operations on it
 
 @centos_minion
-  Scenario: Delete the CentOS SSH minion before normal minion tests
-    When I am on the Systems overview page of this "ceos_ssh_minion"
+  Scenario: Delete the CentOS minion before SSH minion tests
```

Comment:
I wonder why we delete the centos minion, to test centos ssh minion, apparently, we are not reusing the same VM :thinking:

Reply:
We are using two tags, but currently the VMs are the same, that's why we need to delete in that step.
See here:

`testsuite/features/support/twopence_init.rb`
```
$node_by_host = { 'server'                => $server,
                  'proxy'                 => $proxy,
                  'ceos_minion'           => $ceos_minion,
                  'ceos_ssh_minion'       => $ceos_minion,
                  'ceos_client'           => $ceos_minion,

```

Reply:
Cool, got confused, all good then